plugins/omemo/src/ui/contact_details_dialog.vala

```diff
@@ -48,7 +48,7 @@ public class ContactDetailsDialog : Gtk.Dialog {
         automatically_accept_new_descr.label = _("When this contact adds new encryption keys to their account, automatically accept them.");
         own_key_label.label = _("Own key");
         new_keys_label.label = _("New keys");
-        associated_keys_label.label = _("Associated keys");
```

Comment:
I see, I wasn't aware of it. The problem here is then that static UI elements (titles/labels) are dynamically changed which isn't a recommended practice. The difficulty to find wording which works in every case is one reason why it's not recommended, the difficulty to explain the situation to translators is another and there are some more.

So you could always show the three suggested labels ("own", "new" and "other") and just leave the list for the second one empty if there are no new keys. Lists are dynamic elements so as long as it's visible that it's an empty list, that's a valid solution.

If you don't want that, you could name the last one "Acknowledged keys" because as I understand the list shows keys which have been processed in some way. Either automatically/manually accepted or verified, and I assume rejected keys would be there as well.